.github/workflows/package_constraint_solver.yaml

```diff
@@ -1,9 +1,13 @@
 name: Package constraint_solver tests
 on:
   push:
-    branches: [ "main" ]
```

Comment:
For the time being, maybe just edit this to `branches: ["master"]` until this repo gets a `main` branch

Reply:
I'm going to keep this as-is so it's future-proofed.